### DISCUSSION TITLE
Issue with flickering screen on Ubuntu

### DISCUSSION THREAD

**Discussion by mobrembski** [i3/i3#5397] [Q&A]
Hi all!
After switching to new laptop i'm facing a little bit weird issue. Well, maybe i will share a video about issue, for better understanding:
[https://streamable.com/1xc8af](veed file share)

You might think that i'm pressing esc so dmenu dissapers. But it's not. Its look like  my system "created" a snapshot of my screen and for a second displayed it instead of real display ;)
I'm experiencing this in every application, and i feel it might be some related on inputs? Since i mostly have this issue while being in input control when system is waiting to some keystrokes.

You might tell me that it is a graphics driver related issue, but unfortunately it's not ;( I'm using Ubuntu 22.10, and my laptop (Dell Prestige 5585) has Nvidia Quadro P2000 graphic card. I've tried different drivers (open/free, and proprietary ones), also standard ubuntu Gnome with Wayland and with Xorg. And one common thing is that this issue is reproductible ONLY on i3 :(

I had no idea where to find help, what to test. Please give me some advice.

**Comment by claf** [comment]
Hi, I've just made a post before realizing you were having the same issue as me. Only difference is that my issue seems to only occurs when I have an app in fullscreen. (#5413 )

**Reply by mobrembski** [reply]
@claf what graphics driver you use?

I have Nvidia prime dual GPU - Nvidia and Intel UHD. Issue only occurs when i use on-demand or Intel GPU only, it's not reproductible when PC is forced to use only Nvidia. So so far workaround for me is to use NVidia only.
However it's not a solution, My notebook doesnt want to go sleep and has terrible power usage when using Nvidia.

Still, on original Gnome from Ubuntu issue doesnt reproduces even if using on-demand or Intel GPU.

**Reply by claf** [reply]
> @claf what graphics driver you use?
> 
> I have Nvidia prime dual GPU - Nvidia and Intel UHD. Issue only occurs when i use on-demand or Intel GPU only, it's not reproductible when PC is forced to use only Nvidia. So so far workaround for me is to use NVidia only. However it's not a solution, My notebook doesnt want to go sleep and has terrible power usage when using Nvidia.
> 
> Still, on original Gnome from Ubuntu issue doesnt reproduces even if using on-demand or Intel GPU.

I don't have anything but the integrated GPU : Intel(R) UHD Graphics.

BTW your video is not available anymore (appears as archived and not been able to watch it). So the issue occurs regardless of your app being in fullscreen or not, so it's close to unusable as is? My workaround is pretty obvious, I don't put apps in fullscreen.

**Reply by mobrembski** [reply]
@claf I have reuploaded it again:
[https://streamable.com/m77kqv](https://streamable.com/m77kqv)

Asyou might see, in my case it doesnt matter if i turn on full creen or not.
And at least we found common place where this issue happens - Intel UHD drivers. Mine problem reproduces only on Intel GPU, and you have only Intel GPU.

**Reply by claf** [reply]
Again, the video is marked as already archived :) one has to be pretty quick to view it!

**Reply by mobrembski** [reply]
@claf  Haha :) i've registered on streamable, should be available now for good
[https://streamable.com/zm5lqx](https://streamable.com/zm5lqx)

**Reply by claf** [reply]
Ok so it clearly looks like my issue too (although as I said, I never have this issue if I don't have an app in fullscreen)

**Reply by mobrembski** [reply]
It's definitely an issue of Intel UHD drivers, but currently i cant find on google any information how to solve it

**Reply by claf** [reply]
But still somehow related to i3 since I'm pretty sure I never faced such a behavior with previous WM.
I have no idea what to look for though, really hope some folks around here can point in the right direction.

**Reply by mobrembski** [reply]
@claf  Have you tried this:
https://www.youtube.com/watch?v=cGbIPFDZoOU
?
I havent yet but reddit folks in i3wm subreddit tells that it could help

**Reply by claf** [reply]
I'll let you know as soon as I restart my laptop if it helps.

**Reply by claf** [reply]
Well so far it really looks like it's solving the issue for me!

**Reply by claf** [reply]
Did you check if it solves the issue for you @mobrembski ?

**Reply by mobrembski** [reply]
Today i had finally chance to test it.
It looks like it helps, however i'm missing now HW acceleration on Chrome. Need to check this ;)
